benchmarks/utilities.py

```diff
@@ -0,0 +1,32 @@
+"""Utilities for benchmark tests."""
+
+import logging
+import json
+from rules_python.python.runfiles import runfiles
+import os
+from shutil import copyfile
+
+
+def output_benchmark_results(json_results, fixture):
+  """Output the benchmark results."""
```

Comment:
Can we expand this and document the arguments?

We could also add a few more details into the docstring indicating what this utility does, since it isn';t immediately obvious.

Reply:
done.